test/e2e/kubeTest.sh

```diff
@@ -0,0 +1,162 @@
+# Copyright 2017 Istio Authors
```

Comment:
This is awesome. Since it tests integration between all modules, should we move it to istio module then ?